### ISSUE TITLE
Merge the pipeline and shader views (or similar)

### ISSUE THREAD

**Issue by AWoloszyn** [google/gapid#3090]
Depends on #3089.

Once the pipeline view is in place, it should be able to show a lot of the same information as the old shader/program view. We should figure out how much/what of this we want merged into a single view.

**Comment by AWoloszyn** [comment]
Going to re-phrase these. Closing will open new bugs later.
